client/src/components/organisms/DeleteRequestGroupDialog.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from "react";
+
+import FormModal from "./FormModal";
+
+interface Props {
+    requestGroupName?: string;
+    handleClose: () => void;
+    onSubmit: (e: React.FormEvent) => void;
+    onCancel: () => void;
+    numRequests: number;
+}
+
+const DeleteRequestTypeDialog: FunctionComponent<Props> = (props: Props) => {
+    return (
+        <FormModal
+            className="delete-request-type-form"
```

Comment:
Nit: Makes sense that we're recycling styling code but maybe generalizing that into its own file or smth so the naming makes more sense

Reply:
Done and created new .scss file for this component